hack/update-codegen-dockerized.sh

```diff
@@ -173,15 +129,4 @@ $GOPATH/bin/go-to-protobuf \
   --go-header-file hack/boilerplate/license_header.go.txt
 rm -rf /tmp/includes
 
-set +x
-
-echo "=== Start resetting changes introduced by YEAR ==="
-# The call to 'tac' ensures that we cannot have concurrent git processes, by
-# waiting for the call to 'git diff  --numstat' to complete before iterating
-# over the files and calling 'git diff ${file}'.
-git diff  --numstat | awk '$1 == "1" && $2 == "1" {print $3}' | tac | while read file; do
-  if [[ "$(git diff ${file})" == *"-// Copyright "*" Antrea Authors"* ]]; then
-    git checkout HEAD -- "${file}"
-    echo "=== ${file} is reset ==="
-  fi
-done
+reset_year_change
```

Comment:
maybe you should just have `source hack/update-mockgen-dockerized.sh` at the end? right now `reset_year_change` would be invoked twice I think

Reply:
good point, let me refine it. thanks.

Reply:
done